.travis.yml

```diff
@@ -0,0 +1,61 @@
+language: php
+sudo: required
+php:
+  - 5.6
+  - 7.0
+  - 7.1
+  - 7.2
+
+env:
+  global:
+    - CORE_BRANCH=master
+    - APP_NAME=analysis_app
+  matrix:
+    - DB=sqlite
+
+branches:
+  only:
+    - master
+
+before_install:
+  - which php
+  - more $(which pecl)
+  - which phpize
+  - bash tests/.travis/before_install.sh $APP_NAME $CORE_BRANCH $DB
+  - make
+  - make appstore
+  - cd ../core
+  - php occ app:enable $APP_NAME
+
+script:

```

Comment:
I dont actually know how to give exact test path and how to execute travis ci tool, so it would be nice to discuss this

Reply:
No more babysitting, just spend time a bit, try to find yourself.